sdk/nodejs/runtime/resource.ts

```diff
@@ -593,6 +598,13 @@ async function prepareResource(
 
             resolveURN = (v, err) => {
                 if (err) {
+                    if (isGrpcError(err)) {
+                        if (!shownResolveURNWarning) {
+                            console.error("info: skipped rejection in resolveURN");
```

Comment:
What if I don't want these messages written out from my program that uses automation API? How would I stop it?

Reply:
Naively, I'd assume I could catch any errors and decide if I want to show it or not. Is that not possible?